descriptors/ModuleDescriptor-template.json

```diff
@@ -199,7 +199,17 @@
   ],
   "permissionSets": [
     {
-      "permissionName": "gobi.item.post",
+      "permissionName": "gobi.validate.item.get",
+      "displayName": "gobi - post order",
+      "description": "Creates an order"
```

Comment:
Display names and descriptions are same for new perm sets

Reply:
Fixed